[2023-11-15T13:20:55.694+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_github_to_gcs_dag/run_id=scheduled__2023-11-14T00:00:00+00:00/task_id=bigquery_external_table_task permission to 509
[2023-11-15T13:20:55.790+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_github_to_gcs_dag/run_id=scheduled__2023-11-14T00:00:00+00:00/task_id=bigquery_external_table_task permission to 509
[2023-11-15T13:20:55.828+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_github_to_gcs_dag.bigquery_external_table_task scheduled__2023-11-14T00:00:00+00:00 [queued]>
[2023-11-15T13:20:55.840+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_github_to_gcs_dag.bigquery_external_table_task scheduled__2023-11-14T00:00:00+00:00 [queued]>
[2023-11-15T13:20:55.841+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-15T13:20:55.859+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2023-11-14 00:00:00+00:00
[2023-11-15T13:20:55.870+0000] {standard_task_runner.py:57} INFO - Started process 284 to run task
[2023-11-15T13:20:55.873+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_github_to_gcs_dag', 'bigquery_external_table_task', 'scheduled__2023-11-14T00:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/data_github_to_gcp.py', '--cfg-path', '/tmp/tmp01hwuhcc']
[2023-11-15T13:20:55.876+0000] {standard_task_runner.py:85} INFO - Job 86: Subtask bigquery_external_table_task
[2023-11-15T13:20:55.893+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-11-15T13:20:55.953+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_github_to_gcs_dag/run_id=scheduled__2023-11-14T00:00:00+00:00/task_id=bigquery_external_table_task permission to 509
[2023-11-15T13:20:55.957+0000] {task_command.py:416} INFO - Running <TaskInstance: data_github_to_gcs_dag.bigquery_external_table_task scheduled__2023-11-14T00:00:00+00:00 [running]> on host 2952db3161f1
[2023-11-15T13:20:56.085+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_github_to_gcs_dag' AIRFLOW_CTX_TASK_ID='bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-14T00:00:00+00:00'
[2023-11-15T13:20:56.087+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2023-11-15T13:20:56.088+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-11-15T13:20:56.090+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-11-15T13:20:56.808+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_github_to_gcs_dag, task_id=bigquery_external_table_task, execution_date=20231114T000000, start_date=20231115T132055, end_date=20231115T132056
[2023-11-15T13:20:56.850+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-15T13:20:56.892+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
